examples/python/sample-job.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import argparse
+from kubernetes import config, client
+
+# create_job.py
+# This example will demonstrate full steps to submit a Job.
+
+# Make sure your cluster is running!
+config.load_kube_config()
+crd_api = client.CustomObjectsApi()
+api_client = crd_api.api_client
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Submit Kueue Job Example",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "--job-name",
+        help="generateName field to set for job",
+        default="sample-job-",
+    )
+    return parser
+
+
+def generate_job_crd(job_name):
+    """
+    Generate an equivalent job CRD to sample-job.yaml
+    """
+    metadata = client.V1ObjectMeta(
+        generate_name=job_name, labels={"kueue.x-k8s.io/queue-name": "user-queue"}
+    )
+
+    # Job container
+    container = client.V1Container(
+        image="gcr.io/k8s-staging-perf-tests/sleep:v0.0.3",
```

Comment:
it would be cool to get this and the queue name from params :) And maybe the requests?
But it could be a follow up.

Reply:
Added - also added a parameter for `--args`. What do you  mean by requests?

Reply:
Oh resources:

```
resources={
            "requests": {
                "cpu": 1,
                "memory": "200Mi",
            }
        }
```
I think for an example, this hard coded low value is reasonable. This isn't intended to be a submission script to actually use, but rather just provide an example to customize.

Reply:
right, we don't need to bloat the script :)